frontend/packages/dev-console/integration-tests/features/topology/topology_pipeline_feature-ODC-3925

```diff
@@ -0,0 +1,29 @@
+Feature: Pipeline features in topology
+	As a user, I want to check pipeline features in topology
+
+Background:
+    Given user is at developer perspecitve
+    And user has installed Openshift Pipeline Operator
+    And user has selected namespace "aut-topology-pipeline"
+
+Examples:
+    | resource_type     | workload_name   |
+    | deployment        | nodejs-ex-git-1   |
+    | deployment config | dancer-ex-git-1 |
+
+
+@regression
+Scenario Outline: Sidebar for workload
+   Given user created workload "<workload_name>" with resource type "<resource_type>" with pipeline 
+   When user clicks on workload "<workload_name>"
+   Then user can see sidebar opens with Resources tab selected by default
+   And user can see Pipeline Runs section
+   And user is able to see Start button to start a pipeline run
```

Comment:
Please change the scenario based on the below pointers.

1. When a pipeline options is checked in during the Import flow, then in the topology sidebar user should be able to see then pipelinerun section but the buildconfig section should **NOT** be available.
2. During the +Add flow creation, if the pipeline is not able to auto-start for some reason, only then we will be showing the start button  in the sidebar.